src/views/layouts/includes/f-item-header-content.html

```diff
@@ -0,0 +1,12 @@
+{{#if data.labels}}
+	<ul class="f-Item-labels f-u-listInline">
+		{{#each data.labels}}
+			<li><span class="f-Label f-Label--{{this}}">{{this}}</span></li>
```

Comment:
How does `f-Label` look if `f-Label-something` is undefined (i.e. someone uses a label identifier that in front matter that doesn't map to a defined label classname)?